index.html

```diff
@@ -0,0 +1,64 @@
+<html>
+<head>
+  <title>The Lorem Micro Blog</title>
+  <link rel="stylesheet" type="text/css" href="styles.css"/>
+</head>
+<body>
+  <nav>
+    <h1>
+      The Lorem Micro Blog
+    </h1>
+  </nav>
+  <section id="active-page">
+    <aside>
+      <section class="advertisement">
+        <img src="img/onetip.png" width="200px" alt="terrible 'one weird trick' advertisement">
+      </section>
+    </aside>
+    <main>
+      <header>
+        <h1>The Lorem Micro Blog</h1>
+        <h2>by Foo Bar</h2>
+      </header>
+      <article>
+        <h1> Most Posty Post</h1>
+        <h2>Written 1/3/2000</h2>
+        <h3>First Thoughts</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.</p>
+
+        <h3>Additional Thoughts</h3>
+        <p>Duis aute irure dolor in reprehenderit in voluptate velit esse
+        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+        proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+      </article>
+      <article>
+        <h1>Another Posty Post</h1>
+        <h2>Written 1/2/2000</h2>
+        <h3>Only Thoughts...</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.</p>
+      </article>
+      <article class="wide">
+        <h1>A Posty Post</h1>
+        <h2>Written 1/1/2000</h2>
+        <h3>Be It Resolved</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.</p>
+        <p>Duis aute irure dolor in reprehenderit in voluptate velit esse
+        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+        proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+      </article>
+      <footer>
```

Comment:
If the footer went the entire width of the page instead of just under the articles, would you have put everything under <main> and kept it here or just moved it outside of (and below) main to span the full page?